js/src/components/elements/select.js

```diff
@@ -32,7 +32,7 @@ export default class Select extends Component {
     const { label, data } = this.props;
     return [
       label !== '' ? <label key={`select-label-${label}`} htmlFor={`select-${label}`}>{label}</label> : '',
-      <select key={`select-${label || 'select'}`} multiple size="7" onChange={this.changeHandler} value={Array.from(this.selected)}>
+      <select key={`select-${label || 'select'}`} multiple size="7" onClick={this.changeHandler} value={Array.from(this.selected)}>
```

Comment:
Since we are changing this to `onClick`, how about we change the name of the prop passed to `changeHandler`?

Reply:
Good idea!